lib/serializer.js

```diff
@@ -144,7 +144,14 @@ function parseRelations (data, relations, options) {
     var fk = data[fkName];
 
     var fromType = utils.pluralForModel(relation.modelFrom);
-    var toType = utils.pluralForModel(relation.modelTo);
+    var toType = '';
+    if (relation.polymorphic) {
+      var discriminator = relation.polymorphic.discriminator;
+      var model = options.app.models[data[discriminator]];
+      toType = utils.pluralForModel(model);
+    } else {
+      toType = utils.pluralForModel(relation.modelTo);
+    }
```

Comment:
Totally unsure on that one - in the other project I'm explicitly excluding it in the model definition. I'll check how its handled by the default adapter.